elevate-user/router/index.js

```diff
@@ -1,10 +1,14 @@
 const { passThroughRequester } = require('../utils/requester')
 const { orchestrationHandler } = require('../controllers/orchestrationController')
+const { customHandler } = require('../controllers/customHandler')
 const packageRouter = async (req, res, responses) => {
-	const response = req.orchestrated
-		? await orchestrationHandler(req, res, responses)
-		: await passThroughRequester(req, res)
-	return response
+	const response = req.orchestrated 
+  ? await orchestrationHandler(req, res, responses)
```

Comment:
Don't chain ternary operators like this if it is deeper than one level. Use if elses.